apps/baseline/viewsets.py

```diff
@@ -155,6 +156,21 @@ class Meta:
             *translation_fields("name"),
         )
 
+    country = MultiFieldFilter(
```

Comment:
I think we need to add `country` as a specific field to `LivelihoodZoneFilterSet` with the same fields, and add a unit test. At the moment we can seach the Baselines for Niger, say, but we can only search the Zones using NE

Reply:
Using the `country__iso3166a2` in the `fields` tuple for `LivelihoodZoneFilterSet` doesn't seem to work as complex chain of lookups isn't supported in the fields, so added the a `MultiFieldFilter` declaration like the rest